src/common/direct-link-query.js

```diff
@@ -1,24 +1,35 @@
 'use strict';
 
 /**
+ * Subset of the client configuration which causes the client to show a
+ * particular set of annotations automatically after it loads.
+ *
+ * See https://h-client.readthedocs.io/en/latest/publishers/config/#config-settings
+ *
  * @typedef {Object} Query
- * @property [string] id
- * @property [string] query
+ * @property {string} [annotations] - ID of the direct-linked annotation
```

Comment:
The property name changed from `id` to `annotations` here so that the resulting object can be merged directly into the client's configuration. This avoids the rest of the extension needing to know about the details of direct linking, beyond whether there is a direct link or not.